src/components/Text/index.tsx

```diff
@@ -15,6 +15,10 @@ const textStyles = {
   body: { size: "xl", weight: "normal" },
   "body-small": { size: "lg", weight: "normal" },
   subtext: { size: "base", weight: "normal" },
+  aksaraHeader: { size: "4xl", weight: "extrabold" },
```

Comment:
Can avoid naming new text styles with `aksara` prefixed
Instead, you can rename the whole text styles to something like body, body-alt, header, header-alt, etc.

This will require some files to be changed as they specify the text style, so will need your help with that!